src/app/project/qgsprojectelevationsettingswidget.cpp

```diff
@@ -241,6 +244,19 @@ bool QgsProjectElevationSettingsWidget::validate()
       mMessageBar->pushMessage( tr( "An elevation layer must be selected for a mesh terrain" ), Qgis::MessageLevel::Critical );
     }
   }
+
+  // show an error message if the lower value is greater than the upper one
+  // also show the error message if one of the values is not set
```

Comment:
```
  // also show the error message if one of the values is not set
```
why so?
Can't we have a project with just a minimum or maximum limit?
Even if we couldn't, the error message would be misleading!

Reply:
I have removed the constraint that both limits need to be set.